validation-test/compiler_crashers_2_fixed/0166-sr8160.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: not %target-typecheck-verify-swift
```

Comment:
I’d prefer it if you incorporated this test case into an existing file in test/decl/class/ somewhere.

Reply:
@slavapestov Thanks, I wasn't sure which to favour in this case. I've gone ahead and put the test case in test/inherit/inherit.swift, as that's where the existing "superclass '%0' must appear first in the inheritance clause" diagnostic is currently being tested – I trust this is okay?